src/schemas/user.js

```diff
@@ -4,9 +4,20 @@ const faker = require('faker')
 const uniqueValidator = require('mongoose-unique-validator')
 
 const userSchema = new mongoose.Schema({
-  fullName: {
+  firstName: {
     type: String,
-    required: true
+    required: true,
+    default: ''
+  },
+  middleName: {
+    type: String,
+    require: true,
+    default: ''
+  },
+  lastName: {
+    type: String,
+    require: true,
```

Comment:
@RomuloLlabresJr it seems this is "required".. please also the other fields

Reply:
done updating the code. it is fixed now. thanks